### ISSUE TITLE
Position Control

### ISSUE THREAD

**Issue by OccultSlolem** [Nuts-Bolts-FRC-7528/FRC-2020#8]
Parameters:

- If FMS _has_ given a position control color, than we know that stage 3 is at capacity and we need to do position control (ie we can use one button for both position AND rotation control)
- Keep spinning until color sensor reads the same color as the position determined by the FMS

```
if(PositionControlColor.length() > 0) { //FMS has distributed a position (ie position control is unlocked)
    PositionControl();
}
```

**Comment by OccultSlolem** [comment]
Awaiting further discussion on 2/3

**Comment by OccultSlolem** [comment]
Merged into #17
